src/web/layouts/CommentLayout.tsx

```diff
@@ -78,13 +90,17 @@ const StandardGrid = ({
                         1px /* Vertical grey border */
                         1fr /* Main content */
                         300px; /* Right Column */
+
                     grid-template-areas:
                         'title      border  headline    right-column'
                         'lines      border  headline    right-column'
                         'meta       border  standfirst  right-column'
                         'meta       border  media       right-column'
                         '.          border  body        right-column'
                         '.          border  .           right-column';
+
+                    /* Override grid-template-areas if showcase */
+                    ${display === Display.Showcase ? showcaseGrid : ''}
```

Comment:
Worth seperating into a standard grid and a showcaseGrid rather than rely on an override?

Reply:
agreed 👍